cmd/state-svc/internal/deprecation/deprecation.go

```diff
@@ -0,0 +1,187 @@
+package deprecation
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ActiveState/cli/internal/constants"
+	constvers "github.com/ActiveState/cli/internal/constants/version"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/graph"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/multilog"
+	"github.com/hashicorp/go-version"
+	"github.com/patrickmn/go-cache"
+)
+
+const (
+	// DefaultTimeout defines how long we should wait for a response from constants.DeprecationInfoURL
+	DefaultTimeout = time.Second
```

Comment:
This timeout should be WAY higher now that we don't care about how long it takes. In fact I think we should be using the retry http client.